sigmadsp/communication/sigma_tcp_server.py

```diff
@@ -216,17 +24,19 @@ class SigmaStudioInterface:
     It creates a TCP server, which SigmaStudio talks to.
     """
 
-    def __init__(self, host: str, port: int):
+    def __init__(self, host: str, port: int, dsp_type: str = "adau14xx"):
```

Comment:
I don't think the DSP type should be optional.

Reply:
same as before, I went for "change things without breaking things"; since we already require changes to the config, I'll make this mandatory